specials/SPC18_Matias_SpiesVsMercs.Altis/victory_handler.sqf

```diff
@@ -5,13 +5,22 @@ if (spies_extracted == true) then
 	[mission_victoryHandlerPFH] call CBA_fnc_removePerFrameHandler;
 };
 
-//This will count the alive players
-_alivePlayers = {isPlayer _x && alive _x && side _x == west} count allUnits;
+//This will count the alive spies
+_aliveSpies = {isPlayer _x && alive _x && side _x == west} count allUnits;
```

Comment:
All local variables should have `private`, `allPlayers` can be used here instead of `allUnits`
```sqf
private _spies = { alive _x && { side _x == west }} count allPlayers
```